src/Engine/Data/Texture.cpp

```diff
@@ -132,7 +132,14 @@ void Texture::updateData( const void* data ) {
     } break;
     case GL_TEXTURE_CUBE_MAP: {
         // Load the 6 faces of the cube-map
-        void** texels = (void**)data;
+        void* nullTexels[6] { nullptr, nullptr, nullptr, nullptr, nullptr, nullptr };
```

Comment:
Maybe use 
```suggestion
        static const void* nullTexels[6] { nullptr, nullptr, nullptr, nullptr, nullptr, nullptr };
```
instead ?

Reply:
moved to #904